doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1564,23 +1593,29 @@ All these operations are performed by the introduction tactical ``=>``,
 whose general syntax is
 
 .. tacn:: @tactic => {+ @i_item }
-   :name: =>
+   :name: => (ssreflect)
    :undocumented:
```

Comment:
For some reason there seems to be a rendering issue at this point.

Reply:
Yup, saw that.